### ISSUE TITLE
Can't find anything in input foler

### ISSUE THREAD

**Issue by Alastair97** [TheLastGimbus/GooglePhotosTakeoutHelper#305]
Firstly, thank you so much for making this tool.

I am experiencing a problem and I am sure this isn't a issue on my end. 

I download the zip from google takeout (just one zip - I am not doing any merging of takeout folders). I then extract it to a drive on my PC.

Using the tool, I am first prompted to select the location of the unzipped directory. I select `I:\Takeout`
 
(Inside this `Takeout` folder is a `Google Photos` directory, and inside there is another directory with my files called `Stuff - 2022 Jan - Aug`. This has various `mp4` and `JPG` files and their equivalent `JSON` files. 

I then select the output folder, I select some empty directory I made earlier - `I:\Takeout output`

Then, after I've selected some options it says:

```
Okay, running... searching for everything in input folder...
...oh :(
...
I couldn't find any D: reasons for this may be:
  - you've already ran gpth and it moved all photos to output -
    delete the input folder and re-extract the zip
  - your Takeout doesn't have any "year folders" -
    visit https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper
    again and request new, correct Takeout
After fixing this, go ahead and try again :)
[gpth quitted :( (code 13) - press enter to close]
```

I am unsure what I am doing wrong. The files haven't previously moved to the output, they are all still in `I:\Takeout\Google Photos\Stuff - 2022 Jan - Aug`
I am unsure what "year folders" are? But I have tried this with multiple Google Takeout downloads. 

I have replicated the steps on my C drive and I have ran the takeouthelper as an Admin. 

Any ideas? Am I missing something really obvious?

**Comment by TheLastGimbus** [comment]
Show me screenshot of what you're selecting while requesting a takeout

**Comment by Alastair97** [comment]
Do you mean this? I'm selecting 1 album and nothing else is selected

![Screenshot 2024-04-08 201645](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/80893225/725fa5da-ef60-4c39-997e-b527de15aefb)
![Screenshot 2024-04-08 201848](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/80893225/bdef085e-a651-486d-a704-80c3f937490d)

**Comment by TheLastGimbus** [comment]
Read the readme. There is a graphical guide i made in ms paint and put my heart to it. Please appreciate it :pleading_face:

**Comment by Alastair97** [comment]
I have read it multiple times and I don't see where I am going wrong?

Is it that I _have_ to select all of the albums? That is the only difference I am seeing

**Comment by TheLastGimbus** [comment]
Oh shit, it does not indeed. You got me there :stuck_out_tongue_winking_eye: 

TL;DR you have to select **all** albums available, and it should work fine. Or at least years that interests you

**Comment by Alastair97** [comment]
Thank you 😊

"Or at least years that interests you" So basically, I must select more than 1 album? But I don't have to select all? 

I will attempt this again tomorrow and let you know. 

I probably have a different use case to other people which is why I only selected one album. I just want one of my albums downloaded and organised in the correct date order. I unfortunately cannot just do this via downloading from Google photos because they strip the metadata so it is impossible to organise in date order.
I then downloaded from takeout and now I just want to find an easy way to merge this metadata (in the json files) so I can finally organise this in the right order.

I appreciate your help, by the way

**Comment by TheLastGimbus** [comment]
Years - year folders. "Phtos from 20xx". They contain photos as in camera roll. Albums are just their names

Gpth needs year folders because that's how it works to get everything into one big folder. It's not perfect for your use case, but most stable for getting everything

**Comment by Alastair97** [comment]
Thank you. after selecting the '"photos from..."  folders I' ve now achieved what I wanted to!

Why does it only work with the "photos from..." folders, I'm curious? Anyway, I think it would be good if you make this more clear in the ReadME! 

I've sent you a small donation for my thanks :)

**Comment by TheLastGimbus** [comment]
> I'm curious

Well, Google Takeout, and thus gpth, is weird, unstable and full of edge cases. What should've be a 15 line python script turned out to be complicated machinery with 3 re-writes and 158 gh issues closed over last 4 years. You can read through some random issues to see what weird stuff googled served us. My most memorable are #14, #59, #46, #8, #22, #136

Meanwhile the goal of gpth is to get as accurate state of your photos as we can - so I made it so that it finds all photos in both albums and 'year folders', links them together, and wishes that if one of them has weird edge case that prevents getting it's datetime correctly, the other one will be good :100:

**Comment by MateSucic** [comment]
I've actually found a workaround to export only one album (or more)!

1 You need to share the album to your 2nd account (which doesn't contain any photos)
2 Save the photos so they appear straight in the account gallery
3 Export all the photos via Google Takeout from the 2nd account

Because you don't have any photos from before, the only photos will be the ones from the album/s
